tests/semantic_analyser.cpp

```diff
@@ -1366,6 +1366,24 @@ TEST(semantic_analyser, call_kaddr)
   test("kprobe:f { kaddr(123); }", 1);
 }
 
+TEST(semantic_analyser, call_percpu_kaddr)
+{
+  test("kprobe:f { percpu_kaddr(\"process_counts\"); }");
```

Comment:
is `process_counts` from host? I don't see it in btf data source

Reply:
Yes, it's from the host. It exists on every Linux system but it's still probably better to have it in data_source.